tests/serial/mapping-rbf-gaussian/helpers.cpp

```diff
@@ -0,0 +1,89 @@
+#ifndef PRECICE_NO_MPI
+
+#include "helpers.hpp"
+#include "testing/Testing.hpp"
+
+#include "mesh/Utils.hpp"
+#include "precice/SolverInterface.hpp"
+#include "precice/impl/SolverInterfaceImpl.hpp"
+
+void testRBFMapping(const std::string configFile, const TestContext &context)
+{
+  using Eigen::Vector3d;
+
+  const double z = 0.3;
+
+  // MeshOne
+  Vector3d coordOneA{0.0, 0.0, z};
+  Vector3d coordOneB{1.0, 0.0, z};
+  Vector3d coordOneC{1.0, 1.0, z};
+  Vector3d coordOneD{0.0, 1.0, z};
+  double   valOneA = 1.0;
+  double   valOneB = 3.0;
+  double   valOneC = 5.0;
+  double   valOneD = 7.0;
+
+  // MeshTwo
+  Vector3d coordTwoA{0.0, 0.0, z}; // Maps to vertex A
+  Vector3d coordTwoB{0.0, 0.5, z}; // Maps to edge AD
```

Comment:
Comments here sounds like this would be a NP mapping?
Are the expected values really 1.0 and 4.0 sharp?

Reply:
ah, ok, but then the test does not really test the support radius, right?

Reply:
No, it tests the configuration.

Reply:
But the configuration is not tested in the sense that we actually test that the correct thing is configured. Instead of a proper mapping one could also do white-box testing, but then this test should probably become a unit test instead.